styles.css

```
@@ -0,0 +1,38 @@
+@import "./Components/Avatar/avatarstyles.css";
```

styles keyword can be removed from every component.
```suggestion
@import "./Components/Avatar/avatars.css";
```